src/wallet/rpc/transactions.cpp

```diff
@@ -728,6 +743,9 @@ RPCHelpMan gettransaction()
                                     "'send' category of transactions."},
                                 {RPCResult::Type::BOOL, "abandoned", /*optional=*/true, "'true' if the transaction has been abandoned (inputs are respendable). Only available for the \n"
                                      "'send' category of transactions."},
+                                {RPCResult::Type::ARR, "wallet_descs", /*optional=*/true, "Only if 'category' is 'received'. List of imported descriptors matching the scriptPubKey of this coin.", {
```

Comment:
In 8976cd89051d262bfed2498de5aa9ed190e3b331 "rpc: output wallet descriptors for received entries in listsinceblock"

Not just imported descriptors.